### ISSUE TITLE
Wax toolbar items are not highlighted for keyboard users

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/Kotahi#187]
Severity: Low

Environment:
Prod and Local

Steps to reproduce:
1. Click on Dashboard
2. Click on New Submission
3. Click on Submit a URL instance.
4. Click on Submit your research object.
5. Press Tab key to navigate to next field

Expected results:
Press the Tab key once to navigate to the next field

Actual results:
To reach from Description field to Evaluation Content field should press 6 times on Tab key

![form]([ATTACHMENT: form.PNG - Original GitLab URL: https://gitlab.coko.foundation/uploads/72a68e65c51fd40674220905fff61289/form.PNG - Please manually upload to GitHub and replace this link.])

--- Attachment Summary ---
[ATTACHMENT: form.PNG - Original GitLab URL: https://gitlab.coko.foundation/uploads/72a68e65c51fd40674220905fff61289/form.PNG - Please manually upload to GitHub and replace this link.]

--- Migration Info ---
*Original GitLab Issue: https://gitlab.coko.foundation/kotahi/kotahi/-/issues/184*
*Created by: sebastianrad1 on Fri, 26 Mar 2021 08:05:03 GMT*
*Closed by: bela on Mon, 12 May 2025 11:20:27 GMT*

**Comment by yannisbarlas** [comment]
*Comment by: vukilelanga on Tue, 25 Feb 2025 12:32:04 GMT*

The toolbar items should be highlighted on tabbing through. See HHMI.

**Comment by yannisbarlas** [comment]
*Comment by: MihailGorceagEN on Mon, 26 Apr 2021 14:25:04 GMT*

This task should be resolved when we will migrate on WAX 2
